sdk/third_party/compiler-rt/README.md

```diff
@@ -0,0 +1,10 @@
+Soft-float support library
+==========================
+
+Files here are copied from compiler-rt, unmodified.
```

Comment:
We could make these file read only to protect them against careless editing.

Reply:
Does that work reliably in git?  We don't do that for other vendored file from LLVM.